### PR TITLE
added display from Monday

### PR DESCRIPTION
**Pull request by kekovina**
Added the ability to display the week from Monday. It was necessary for one project, I share the code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,16 +13,19 @@ const HorizontalCalendar = ({
...
 	isDarkWeekend = true,
+	mondayFirst = false, 
 	platform,
 	onClick,
 }) => {
 	const baseClassNames = getClassName("Card", platform);
 
+	const adays = mondayFirst ? daysMondayFirst() : days() 
+
 	const onItemClick = (e) => {
```
Comment (iPagar) [inline src/index.js]: Get up functions days and daysMondayFirst to avoid TypeError. declaring adays should be lower then them.

**Review by iPagar — CHANGES_REQUESTED** [review]
Amazing! 😲 Just make sure to make changes according to the comment before this gets merged.

File: src/index.js (hunk 1 above)
Comment (kekovina) [reply src/index.js]: Done.

**Review by iPagar — APPROVED** [review]
Great!

**Pull request merged**